database/seeds/SignLanguagesTableSeeder.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class SignLanguagesTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $languages = array(
+            collect(['code' => 'dt_DK', 'text' => 'Dansk tegnsprog']),
```

Comment:
Eller om vi laver en translation for hver sprogkode, så vi kan kalde derfra i stedet for? Så vi kan have dansk tekst på alle sprog og så videre?